interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/MethodBasedServiceInvoker.scala

```diff
@@ -36,49 +33,3 @@ private[nodecompilation] class MethodBasedServiceInvoker(
   }
 
 }
-
-private[nodecompilation] class JavaMethodBasedServiceInvoker(
```

Comment:
I wanted to reduce the amount of fields insied `ComponentDefinitionWithImplementation`. Before the change, we had a `runtimeClass` field ( https://github.com/TouK/nussknacker/pull/5279/files#diff-58b0080efe8208403827e1723f5e4aaf21c96000d26fbaa273813b5bb8597e1fL18 ), which was only to handle situation when someone used java Futures instead of scala Futures in their `Service`. I replaced this solution by postprocessing of sth called `ComponentImplementationInvoker` in the definition extraction step https://github.com/TouK/nussknacker/pull/5279/files#diff-c6993653ab16bde95f32da7219816130d116e961a96926400fc4cee6d1ab1e25R103. It reduced this not necessary noise in our definitions and implementation of component is ready to work with `Interpreter` without any further posprocessing.